cluster/gce/config-default.sh

```diff
@@ -402,3 +402,6 @@ ROTATE_CERTIFICATES="${ROTATE_CERTIFICATES:-}"
 # The number of services that are allowed to sync concurrently. Will be passed
 # into kube-controller-manager via `--concurrent-service-syncs`
 CONCURRENT_SERVICE_SYNCS="${CONCURRENT_SERVICE_SYNCS:-}"
+
+# Flexvolume plugin directory
+VOLUME_PLUGIN_DIR="${VOLUME_PLUGIN_DIR:-/etc/srv/kubernetes/kubelet-plugins/volume/exec}"
```

Comment:
Does this need to be set in config-test.sh as well?

Reply:
Does this need to be set conditionally based on the master / node OS type selected? Previously it was only set for gci, but this sets it unconditionally.

Reply:
This was already set in config-test.sh

Shoot, yes you are right. Reverting this back to the original.